mongo/test/org/immutables/mongo/fixture/SimpleUpdaterTest.java

```diff
@@ -0,0 +1,153 @@
+package org.immutables.mongo.fixture;
+
+import org.bson.types.ObjectId;
+import org.immutables.mongo.types.Id;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import static org.immutables.check.Checkers.check;
+
+public class SimpleUpdaterTest {
+
+    @Rule
+    public final MongoContext context = new MongoContext();
+
+    private ItemRepository repository;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
interesting if it's possible (due to `@Rule`, otherwise I know it's possible) to just initialize repo and remove `@Before`

```java
final ItemRepository repository = new ItemRepository(context.setup());
```
for each test method invocation JUnit creates new instance of the objects and looks cleaner, but it's just my preference.
And also formatting, I don't like to bug people about formatting, I just auto-format it after the fact. (2 spaces indent in this repo etc)